ik_problem: {type: yamlfile, value: $PWD/centauro_base_estimation.stack}
#world_frame_link: {type: string, value: l_sole}
world_from_gz: {type: bool, value: true}
use_imu: {type: bool, value: true}
enable_log: {type: bool, value: false}
rolling_contacts:
  type: map<string>
  value:
    contact_1: wheel_1  # ft -> wheel
    contact_2: wheel_2
    contact_3: wheel_3
    contact_4: wheel_4
filter_damping: {type: double, value: 1.}
filter_omega: {type: double, value: 100.}
contact_attach_thr: {type: double, value: 100.}
contact_release_thr: {type: double, value: 50.}
